shared/constants/hardware-wallets.js

```diff
@@ -1,11 +1,18 @@
 /**
- * Accounts can be instantiated from simple, HD or the two hardware wallet
+ * Accounts can be instantiated from simple, HD or the three hardware wallet
```

Comment:
Maybe just "multiple" instead of "three", in case that more hardware wallets get added?